modules/statics/src/coins.ts

```diff
@@ -1167,6 +1153,26 @@ export const coins = CoinMap.fromCoins([
     BaseUnit.ETH,
     ZKETH_FEATURES
   ),
+  account(
+    'ac3c225e-55a9-4236-b907-a4cccc30a2fd',
+    'bera',
+    'berachain',
```

Comment:
I think we are using this name somewhere in UI, so we should correct this
```suggestion
    'Berachain',
```